pages/guides/_section.yml

```diff
@@ -0,0 +1,5 @@
+categories:
```

Comment:
As I was working on this, it didn't feel quite right specifying Category ordering/weights within individual Docs' frontmatter. The Category order doesn't belong to an individual Doc, it's specific to a Section.

If data that belongs to a Section can be defined within any Doc, it's not obvious where that data lives/should live, and how to deal with potentially conflicting data in the many different places it could exist in.

We ran into a similar issue when it came to defining Groups of docs, and solved it by allowing a Group to be created  by adding `group:` to any `index.mdx` Doc's metadata.

This is similar, but allows a section to be created by adding a `_section.yml`. In the future we could expand what's defined here to include more of the stuff we currently have hardcoded in the app (if we want).

Reply:
Agreed. I like this!